assets/app/views/builds.html

```diff
@@ -8,21 +8,86 @@
+                </span>
+                <span ng-switch-when="imageChange">
+                  Image change from image {{trigger.imageChange.image}}:{{trigger.imageChange.tag}}
+                </span>
+                <span ng-switch-default>{{trigger.type}}</span>
+              </span>
+            </li>
+          </ul>
+        </div>
+        <div style="margin-bottom: 10px;" ng-repeat="build in buildsByBuildConfig[buildConfigName]">
+          <h3>{{build.metadata.name}}</h3>
+          <div>Created: <relative-timestamp timestamp="build.metadata.creationTimestamp"></relative-timestamp></div>            
+          <div>Status: {{build.status}}</div>
+          <div ng-if="buildConfig.parameters.strategy.type != build.parameters.strategy.type">
+            <div>Build strategy: {{build.parameters.strategy.type}}</div>
+            <div ng-if="build.parameters.strategy.type == 'STI'">
+              Builder image: {{build.parameters.strategy.stiStrategy.builderImage}}
+            </div>
+          </div>
+          <div ng-if="build.parameters.source">
+            <div ng-if="buildConfig.parameters.source.type != build.parameters.source.type">
+              <div ng-if="build.parameters.source.type == 'Git'">Source repo: {{build.parameters.source.git.uri}}</div>
+            </div>
+            <!-- TODO add git ref details -->
+          </div>
+          <div ng-if="buildConfig.parameters.output.to.kind != build.parameters.output.to.kind && buildConfig.parameters.output.to.name != build.parameters.output.to.name">
+            <div ng-switch="build.parameters.output.to.kind">
+              <div ng-switch-when="ImageRepository">
+                Output image: {{buildConfig.parameters.output.to.name}}
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <!-- render any builds whose build configs no longer exist -->
+        <div style="margin-bottom: 10px;" ng-repeat="build in builds" ng-if="!buildConfigs[build.metadata.labels.buildconfig]">
+          <h3>{{build.metadata.name}}</h3>
+          <div>Created: <relative-timestamp timestamp="build.metadata.creationTimestamp"></relative-timestamp></div>            
+          <div>Status: {{build.status}}</div>
+          <div>Build strategy: {{build.parameters.strategy.type}}</div>
+          <div ng-if="build.parameters.strategy.type == 'STI'">
+            Builder image: {{build.parameters.strategy.stiStrategy.builderImage}}
+          </div>
+          <div ng-if="build.parameters.source">
+            <div ng-if="build.parameters.source.type == 'Git'">Source repo: {{build.parameters.source.git.uri}}</div>
+            <!-- TODO add git ref details -->
+          </div>
+          <div ng-switch="build.parameters.output.to.kind">
+            <div ng-switch-when="ImageRepository">
+              Output image: {{buildConfig.parameters.output.to.name}}
```

Comment:
For a build, the output.DockerImageReference is likely more relevant. If you specify output.to in the buildConfig, when a build is actually created, the output.to is resolved to a specific output.DockerImageReference